### PR TITLE
ci: fix android java wrapper generation with web3j

### PR DESCRIPTION
**Pull request by CJ42**
# What does this PR introduce?

## 🤖 CI

Fix the Web3j issue related to generating the Java wrapper to create the android artifacts.

This is related to: https://github.com/web3j/web3j/issues/1855

### CODE REVIEW

File: scripts/ci/generate-java-wrapper.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+abi_dir=( ./android/abi/* )
+bin_dir=( ./android/bin/* )
+n=${#abi_dir[@]}               # assume they're the same length
+for (( i = 0; i < n; i++ )); do
```
Comment (richtera) [inline scripts/ci/generate-java-wrapper.sh:4]: An indexes list, seems complicated.

File: scripts/ci/generate-java-wrapper.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+abi_dir=( ./android/abi/* )
```
Comment (richtera) [inline scripts/ci/generate-java-wrapper.sh:1]: I think there should be a shebang.

**Review by richtera — CHANGES_REQUESTED** [review]
Not sure about the shell script.

**Review by richtera — APPROVED** [review]
LGTM

**Review by JeneaVranceanu — APPROVED** [review]

**Review by b00ste — APPROVED** [review]

**Pull request merged**